src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+        for (int i = 0; i < Task.getNoOfTask(); i++) {
+            if(t[i].getTaskType().equals("Deadline")) {
+                System.out.println("     " + (i+1) + "." + t[i]);
+            } else if(t[i].getTaskType().equals("Events")) {
+                System.out.println("     " + (i+1) + "." + t[i]);
+            } else {
+                System.out.println("     " + (i+1) + "." + t[i]);
+            }
+        }
+        System.out.println("    ____________________________________________________________\n");
+    }
+
+    public static void taskMarkAsDone(int itemIndex) {
+        String taskDoneDescription = "";
+        for(int i = 0; i < Task.getNoOfTask(); i++) {
+            if(itemIndex == i) {
+                taskDoneDescription = t[i].getTaskDescription();
+                t[i].taskDone();
+            }
+        }
+        System.out.println("    ____________________________________________________________\n" +
+                "     Nice! I've marked this task as done:\n" +
+                "       [✓] " + taskDoneDescription + "\n" +
+                "    ____________________________________________________________\n");
+    }
+
+    public static void addTask(String currentInput) {
+        String taskType = currentInput.split(" ")[0];
+        String taskDescription;
+
+        if(currentInput.split(" ").length > 1) {
+            taskDescription = currentInput.split(" /", 2)[0].split(" ", 2)[1];
+        }
+        else {
+            taskDescription = currentInput;
+        }
+
+        switch (taskType) {
+        case "deadline":
+            String deadlineBy = currentInput.split("/by ")[1];
+            t[Task.getNoOfTask()] = new Deadline(taskDescription, deadlineBy);
+            break;
+        case "event":
+            String eventAt = currentInput.split("/at ")[1];
+            t[Task.getNoOfTask()] = new Events(taskDescription, eventAt);
+            break;
+        case "todo":
+            t[Task.getNoOfTask()] = new ToDo(taskDescription);
+            break;
+        default:
+            t[Task.getNoOfTask()] = new ToDo(currentInput);
+            break;
+        }
+        System.out.println("    ____________________________________________________________\n" +
+                "     Got it. I've added this task:\n" +
+                "       " + t[Task.getNoOfTask()-1] + "\n" +
+                "     Now you have " + (Task.getNoOfTask()) + " tasks in the list.\n" +
+                "    ____________________________________________________________\n");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greetings = "    ____________________________________________________________\n" +
```

Comment:
I would suggest if you could create this into a function as well as it will improve the readability